scripts/fetch-develop.deps.sh

```diff
@@ -40,9 +40,8 @@ function dodep() {
     # Try the PR author's branch in case it exists on the deps as well.
     # Try the target branch of the push or PR.
     # Use the default branch as the last resort.
-    if [[ "$TRAVIS" == true ]]; then
-        clone $org $repo $TRAVIS_PULL_REQUEST_BRANCH ||
```

Comment:
It looks like `$BUILDKITE_BRANCH` is the PR author's branch, while `$BUILDKITE_PULL_REQUEST_BASE_BRANCH` is the target branch. I think we can use both to preserve current behaviour here.

Reply:
Whups, good catch. Didn't mean to drop this.